cps/tasks/download.py

```diff
@@ -54,6 +56,9 @@ def run(self, worker_thread):
                 last_progress_time = datetime.now()
                 fragment_stuck_timeout = 30  # seconds
 
+                self.message = f"Downloading {self.media_url_link}..."
+                if self.live_status == "was_live":
+                    self.message += f" (This may take longer than expected due to the video was live for {self.duration})"
```

Comment:
Do we dare abbreviate the message to something like... ?

```suggestion
                    self.message += f" (formerly live video, length/duration is {self.duration} seconds)"
```